projects/experimental/directives/button-group/button-group.directive.ts

```diff
@@ -0,0 +1,15 @@
+import {Directive, Inject} from '@angular/core';
+import {TuiDirectiveStylesService} from '@taiga-ui/cdk';
+
+import {TuiButtonGroupComponent} from './button-group.component';
+
+@Directive({
+    selector: '[tuiButtonGroup]',
+})
+export class TuiButtonGroupDirective {
+    constructor(
+        @Inject(TuiDirectiveStylesService) directiveStyles: TuiDirectiveStylesService,
+    ) {
+        directiveStyles.addComponent(TuiButtonGroupComponent);
```

Comment:
you can already use `tuiWithStyles` approach like in `header` or `sensitive`

Reply:
and `standalone` so as not to return later

Reply:
Next PR